dht/dht.go

```diff
@@ -112,7 +128,7 @@ func (nid *nodeID) ByteString() string {
 	return string(buf[:])
 }
 
-type node struct {
+type Node struct {
```

Comment:
This should not be made public. A user of OnQuery (and presumably other future hooks) can build their own own node table if they want. Nodes are transient and should not be visible. gofmt -r 'Node -> node' -w will make this easy to fix, or try gorename.